### DISCUSSION TITLE
Should I use the RNA assay or SCT assay for DE with pseudobulk?

### DISCUSSION THREAD

**Discussion by Zack-G-123** [satijalab/seurat#8373] [Q&A]
I have a Seurat v5 object from 4 samples that were integrated and clustered using the SCTransform pipeline. I would like to try and perform DE testing using pseudobulk samples to see if I can eliminate some sample biases. Should I perform `AggregateExpression()` on the RNA assay, or on the SCT assay with corrected counts obtained from `PrepSCTFindMarkers()`? 

And is it even valid to do so if I want to compare 2 control samples vs 2 experimental, seeing as I'll essentially be performing DESeq2 with n=2 per group?

**Comment by IrenaeusChan** [comment]
Hey Zack-G-123 I was also looking for this answer and I found this post on [biostars](https://www.biostars.org/p/395951/). To summarise:

> For gene-based procedures like differential expression (DE) analyses or gene network construction, it is desirable to use the original log-expression values or counts. The corrected values are only used to obtain cell-level results such as clusters or trajectories. Batch effects are handled explicitly using blocking terms or via a meta-analysis across batches. We do not use the corrected values directly in gene-based analyses, for various reasons:
> 
> It is usually inappropriate to perform DE analyses on batch-corrected values, due to the failure to model the uncertainty of the correction. This usually results in loss of type I error control, i.e., more false positives than expected.
> 
> The correction does not preserve the mean-variance relationship. Applications of common DE methods like edgeR or limma are unlikely to be valid.
> 
> Batch correction may (correctly) remove biological differences between batches in the course of mapping all cells onto a common coordinate system. Returning to the uncorrected expression values provides an opportunity for detecting such differences if they are of interest. Conversely, if the batch correction made a mistake, the use of the uncorrected expression values provides an important sanity check.

Therefore I think for DE with pseudobulk I think it is better to use the RNA counts rather than SCT or integrated counts. 

For your second question
> And is it even valid to do so if I want to compare 2 control samples vs 2 experimental, seeing as I'll essentially be performing DESeq2 with n=2 per group?

While not necessarily "valid" because you should have more replicates to make a truly good DE analyses you can always change the FindMarkers function `min.cells.group = 2`
